_posts/2020-11-30-flexible-and-safe-tuples-in-scala3.md

```diff
@@ -0,0 +1,382 @@
+---
+layout: blog-detail
+post-type: blog
+by: Vincenzo Bazzucchi, Scala Center
+title: Flexible and safe tuples in Scala 3
+---
+
+# Flexible and safe tuples in Scala 3
+
+Tuples are revisited and completely rethought in Scala 3.
+They are more **flexible**, more dynamic and support a **wider range of operations**.
+This is enabled by new and powerful language features.
+
+In this post we will explore the new capabilities of tuples before
+looking under the hood to learn how the improvements in the Scala 3
+type system enable to implement type safe operations on tuples.
```

Comment:
Oh, I just noticed: "enable to implement" doesn't work grammatically, you need either "enable us to implement" or "enable implementing".